deep/permissions.py

```diff
@@ -252,7 +252,9 @@ class Permission(Enum):
         # ---------------------- Lead
         CREATE_LEAD = auto()
         VIEW_ONLY_UNPROTECTED_LEAD = auto()
+        VIEW_ONLY_UNPROTECTED_ASSESSMENT_REGISTRY = auto()
```

Comment:
We don't need VIEW_ONLY_UNPROTECTED_ASSESSMENT_REGISTRY, as this can be derived from VIEW_ONLY_UNPROTECTED_LEAD/VIEW_ALL_LEAD

Basically, we will need VIEW_ASSESSMENT_REGISTRY to check if the user has permission to see assessments and then use VIEW_ONLY_UNPROTECTED_LEAD/VIEW_ALL_LEAD to filter out assessments using the lead's confidential field.

Reply:
Let's use VIEW_ASSESSMENT_REGISTRY instead of VIEW_ALL_ASSESSMENT_REGISTRY